sn_node/src/node/messaging/mod.rs

```diff
@@ -36,9 +37,9 @@ use std::collections::BTreeSet;
 #[derive(Debug, Clone)]
 #[allow(clippy::large_enum_variant)]
 pub(crate) enum OutgoingMsg {
-    System(SystemMsg),
-    Service(ServiceMsg),
-    DstAggregated((BlsShareAuth, Bytes)),
+    Node2Node(Node2NodeMsg),
+    Client(ClientMsg),
+    Consensus((SectionAuthPart, Bytes)),
```

Comment:
I think this will just confuse everyone if we call this elder signature aggregation "consensus" when we have other modules running "consensus protocols".

Can we call it `SectionAuth`?
```suggestion
    SectionAuth((SectionAuthPart, Bytes)),
```